src/Symfony/Component/HttpKernel/Controller/TraceableControllerResolver.php

```diff
@@ -49,18 +49,4 @@ public function getController(Request $request)
 
         return $ret;
     }
-
-    /**
-     * {@inheritdoc}
-     */
-    public function getArguments(Request $request, $controller)
```

Comment:
It's not a BC break as it will duck-type on the default implementation. When this code is released, there will be no implementations without this method so everything will be alright. I just have to convince @fabpot of this ;)